doc/working-stealth-jobs.md

```diff
@@ -1,3 +1,84 @@
 # Working Stealth Jobs
 
-TBD
+**[YearnStealthVault](https://etherscan.io/address/0xYearnStealthVault#code)**
+
+**[YearnStealthRelayer](https://etherscan.io/address/0xYearnStealthRelayer#code)**
+
+
+
+### How to bond:
+
+- to add bonds you need to call (bond) with as much ETH (msg.value) as you'd like to bond.
+    - check amount being bonded is higher than the penalty of the job you want to perform work on
+
+- call `enableStealthContract(address _contract)` with `_contract` being the address of the contract you want to perform work on
+    - in this case `0xYearnStealthRelayer` (YearnStealthRelayer)
+    - you can also use `enableStealthContracts(address[] calldata _contracts)`
+    - this is required since contracts have the ability to take your bond away.
+        - so, please be careful and only enable contracts after reviewing them.
+
+- to remove a contract from your list just call `disableStealthContract(address _contract)`
+    - you can also use `disableStealthContracts(address[] calldata _contracts)`
+
+- to unbond, call (startUnbond), wait 4 days, and call (unbond | unbondAll).
+    - in this period you'll not be able to perform any work
```

Comment:
Is this ok ? do we want this ? I thought, we would just take (`bonded - unbonding` when checking if he has more than needed for the penalty)

Reply:
easier to code, and secure.